docs/api-reference/utility-functions.md

```diff
@@ -347,7 +347,7 @@ Extracts the range spec information `{ step?: number, min?: number, max?: number
 #### Returns
 - RangeSpecType: A range specification from the schema
 
-### schemaRequiresTrueValue()
+### schemaRequiresTrueValue\<S extends StrictRJSFSchema = RJSFSchema>()
```

Comment:
```suggestion
### schemaRequiresTrueValue<S extends StrictRJSFSchema = RJSFSchema>()
```

Reply:
@nickgros the backslash allows the markdown notation to be properly escaped otherwise `<S` renders incorrectly